bin/test

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+require __DIR__ . '/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php';
+
+use CodeIgniter\CLI\CLI;
+use Translations\Tests\AbstractTranslationTestCase;
+
+$phpCsFixer = 'vendor/bin/php-cs-fixer';
+$phpunit    = 'vendor/bin/phpunit';
+if (CLI::isWindows()) {
+    $phpCsFixer = strtr($phpCsFixer, '/', '\\');
+    $phpunit    = strtr($phpunit, '/', '\\');
+}
+
+CLI::write('Running `composer update`...', 'black', 'green');
+passthru('composer update');
+CLI::write();
+
+CLI::write('Fixing coding style...', 'black', 'green');
+passthru(
+    "{$phpCsFixer} fix --verbose --diff"
+);
+CLI::write();
+
+$testCase = new class () extends AbstractTranslationTestCase {
+    public function getTestCaseClassname(string $locale): string
+    {
+        foreach (self::$locales as $classname => $localCode) {
+            if ($localCode === $locale) {
+                $namespaces = explode('\\', $classname);
+
+                return end($namespaces);
+            }
+        }
```

Comment:
Instead of iterating over the `$locales` property, we can `array_flip` it and obtain the class name, or `null` when absent. Then, if `null` exit immediately.
```php
$class = array_flip(self::$locales)[$locale] ?? null;

if ($class === null) {
    CLI::write(...);

    exit(1)
}
```

Reply:
Fixed.